src/scripts/treemacs-git-status.py

```diff
@@ -36,6 +36,8 @@ def find_recursive_entries(path, state):
 
 def main():
     global output, ht_size
+    # Don't lock Git when updating status.
+    environ["GIT_OPTIONAL_LOCKS"] = "0"
```

Comment:
Do we really need both?

Reply:
The Emacs-side change was for the simple Git integration, while this one is for deferred.

Reply:
Ah, right.